src/sentry/runner/initializer.py

```diff
@@ -474,6 +476,20 @@ def monkeypatch_django_migrations():
     monkey_migrations()
 
 
+def monkeypatch_drf_jsonrenderer_encoder_class():
+    # We'd like to keep the default STRICT_JSON=True for DRF parsers,
+    # but rather than error on nan/inf for rendering or pass them through,
+    # we'd like to coerce to null to give to frontend.
+    # It isn't possible with DRF's JSONRenderer, but our simplejson encoder is configured
+    # for this with ignore_nan=True which overrides the allow_nan set by DRF's JSONRenderer
+    # if STRICT_JSON=True.
+    from rest_framework.renderers import JSONRenderer
+
+    from sentry.utils.json import DefaultJSONEncoder
+
+    JSONRenderer.encoder_class = DefaultJSONEncoder
```

Comment:
Is this going to change the format of our responses in a lot of places?

Reply:
I believe the only difference should be that any out-of-range float objects like `float("nan")`, `float("inf")` present in the data to be serialized by DRF for responses will be `"null"` rather than `"nan"` or `"inf"` (behavior before DRF 3.8.x).

And looking at https://sentry.io/organizations/sentry/issues/2378549472/activity/?project=1, it's 86 events to 42 users so I'm confident this only affects internal APIs, so @getsentry/ecosystem shouldn't be worried.

Reply:
Oh, see https://sentry.io/organizations/sentry/issues/2378549472/tags/transaction/?project=1, it's only limited to `/api/0/organizations/{organization_slug}/eventsv2/`. I don't see any other similar issue groups.